frontend/package.json

```diff
@@ -26,6 +26,7 @@
 		"eslint-plugin-jest": "^23.8.2",
 		"eslint-plugin-prettier": "^3.1.3",
 		"eslint-plugin-react": "^7.19.0",
```

Comment:
Do I need to additionaly configure something (except for doing `npm install` and `npm run dev`)?
![image](https://user-images.githubusercontent.com/23579650/93266079-de570500-f7a9-11ea-8446-d864277f3e28.png)